index.js

```diff
@@ -142,16 +141,34 @@ function replaceEnvVars(config) {
     });
 }
 
+function requiresConfig(options) {
+    return !options.generate && (options.start || options.test);
```

Comment:
AFAIK, only `start` and `test` need the config. Neither `generate` nor `build` do.